src/client/results/Feed.js

```diff
@@ -46,32 +46,30 @@ Feed.propTypes = {
   hiddenUsers: React.PropTypes.array,
 };
 
-let PaginationButtons = ({ dispatch, numberOfPages, paginationInfo }) => {
-  return (
-    <div className="ui grid">
-      <div className="two column row">
-        <div className="left column">
-          <LimitButtons paginationInfo={paginationInfo}/> results per page.
-        </div>
-        <br />
-        <div className="right aligned column">
-          <PagePicker numberOfPages={numberOfPages} paginationInfo={paginationInfo} />
-        </div>
+let PaginationButtons = ({ dispatch, numberOfPages, paginationInfo }) => (
+  <div className="ui grid">
+    <div className="two column row">
+      <div className="left column">
+        <LimitButtons paginationInfo={paginationInfo} dispatch={dispatch} /> results per page.
+      </div>
+      <br />
+      <div className="right aligned column">
+        <PagePicker numberOfPages={numberOfPages} paginationInfo={paginationInfo} dispatch={dispatch} />
```

Comment:
I know I should have pickedd this up last PR, but can you change passing down dispatch to just passing down 
`(limit) => {dispatch(setFeedPageLimit(limit));}`